### DISCUSSION TITLE
Set "Search in Data" as default search behavior

### DISCUSSION THREAD

**Discussion by johnr9816** [benjaminjonard/koillection#783] [Ideas]
I would like to see "Search in data (for items and collections)" be set by default, remember its last setting, or have the option to toggle it as being checked by default on searches. 

I have a collection of books as one of the collections and am using a barcode scanner to search by ISBN using its UPC. While in the main search box, a result appears, because the scanner sends the carriage return it goes to the search results page with no results. I then have to check the box to "search in data" and click to search again. However, so long as I do not leave the page after that subsequent scans work fine.

**Comment by benjaminjonard** [comment]
Hello, I'm going to add an option in Settings to search in data by default. It should be done for the next release.

**Reply by johnr9816** [reply]
I see this has now been added. Thanks a lot!
